README.md

```diff
@@ -57,6 +57,37 @@ Once you have installed the credential helper, see the
 [Configuration section](#Configuration) for instructions on how to configure
 Docker to work with the helper.
 
+### Debian Buster (testing) and Debian Sid (unstable)
```

Comment:
I'd try to take a more general approach here. Buster won't be testing forever, and there will be a new testing at some point. This text is thus subject to bitrot. Might be worth instead describing that this package will be included in buster and future releases without assuming specific name mappings.

Reply:
My intent was just to update this when Buster becomes stable.  Do you think that's too complicated?

Reply:
Yeah, seems annoying to have to make commits to this repo just to note activity in third-party projects.

Reply:
Done